.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ env:
   CARGO_NET_RETRY: 10
   CI: 1
   RUST_BACKTRACE: short
-  RUSTFLAGS: -D warnings
+  RUSTFLAGS: -D warnings -C linker=clang -C link-arg=-fuse-ld=lld
```

Comment:
You may need to install lld first.

Reply:
I kinda hoped that it is there by default, but no :(

My endg goal is to try lld on windows (as it is the CI bottleneck). I guess I can do this by seetting different RUSTFLAGS per OS, but I can't figure a convenient way to do that